Readme.md

```diff
@@ -212,6 +212,37 @@ If no provider options are specified by user, OpenVINO™ Execution Provider is
 backend = "CPU"
 precision = "FP32"
 ```
+
+## Code Sample
+
+Below is an example of how you can leverage OpenVINO™ integration with Torch-ORT in a simple NLP usecase. 
+A pretrained BERT model fine-tuned on the CoLA dataset from huggingface model hub is used to predict grammar correctness on a given input text. 
```

Comment:
Can we provide link for hugging face model for quick access?

Reply:
Added